pkg/updater/gcs_test.go

```diff
@@ -1068,7 +1149,12 @@ func TestConvertResult(t *testing.T) {
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
 			log := logrus.WithField("test name", tc.name)
-			actual, err := convertResult(log, tc.nameCfg, tc.id, tc.headers, tc.metricKey, tc.result, tc.merge)
+			if tc.opt == nil {
```

Comment:
nit: I think I'd rather see this as an explicit add to these test cases

Reply:
sure! Working on a follw-up.
/hold cancel